package-lock.json

```diff
@@ -49,7 +49,7 @@
                 "mocha-fivemat-progress-reporter": "^0.1.0",
                 "ms": "^2.1.3",
                 "node-fetch": "^3.3.2",
-                "playwright": "^1.41.0",
+                "playwright": "^1.41.1",
```

Comment:
This change doesn't seem to be related and should be reverted.

Reply:
apologies, I always run `npm up --save` by default